plugin.js

```diff
@@ -60,6 +60,11 @@ function plugin (fastify, options, next) {
   fastify.decorate('parseCookie', function parseCookie (cookieHeader) {
     return cookie.parse(cookieHeader, options.parseOptions)
   })
+
+  fastify.decorate('unsignCookie', function unsignCookie (value) {
```

Comment:
why is `unsignCookie` decorations have 3 different function declaration? Isn't possible to factorize those?

Reply:
I'm not sure what you mean by factorizing the 3 different function declarations, i don't see how building a factory to construct functions for each decorator is really necessary, but i refactored the code to use only one `unsignCookie` function  for each of the decorators.

Reply:
@zekth the others are named according to their usage path for easier debugging. It seems we need this one to support websockets in some way 🙁

Reply:
I wasn't aware of this case. So indeed it make sense also.